### ISSUE TITLE
1.51 Damage text on set bonuses

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#159]
Extra plus if a set items give us bonuses to Damage.
- On normal items
![image](https://user-images.githubusercontent.com/47291794/71150820-78057c00-2243-11ea-8245-6287754f87a8.png)
- On set bonus
![image](https://user-images.githubusercontent.com/47291794/71150900-aaaf7480-2243-11ea-8540-86d170964b4d.png)

**Comment by insthync** [comment]
You can fix it by change codes in `UIBaseEquipmentBonus.cs`

line 191 from 
```
                    entry.amount.max.ToBonusString());
```
To
```
                    entry.amount.max.ToString("N0"));
```

**Comment by insthync** [comment]
This fix included in 1.52
